_pages/user-experience/getting-started.md

```diff
@@ -0,0 +1,80 @@
+---
+title: User Experience
+lead: >
+  Create a simple and consistent experience for your Login.gov users
+sidenav:
+  - text: Getting started
+    href: "user-experience/getting-started/"
+  - text: Sign-in and sign-out buttons
+    href: "user-experience/sign-in-sign-out/"
+  - text: Your agency logo
+    href: "user-experience/agency-logo/"
+  - text: Cancel URL
+    href: "user-experience/cancel-url/"
+  - text: Failure to proof URL
+    href: "user-experience/failure-proof/"
+  - text: Knowledge articles
+    href: "user-experience/knowledge-articles/"
+  - text: FAQ content 
+    href: "user-experience/faq-content/"
+
+---
+
+##  Getting started
+There are several decisions during the integration process that affect how your users experience Login.gov. In this User Experience guide, we will outline what options you have available to your application, and the configurations in Login.gov [dashboard](https://idp.int.identitysandbox.gov/) that impact users.
```

Comment:
this should be the link to dashboard, not the sandbox idp, correct? Also I think you either need a `the` 

```
 and the configurations in the Login.gov [dashboard](https://dashboard.int.identitysandbox.gov/) that impact users
```

Reply:
yep thanks for catching!